<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowWidget</class>
 <widget class="QWidget" name="FlowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="firstName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lastName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="evoButton">
     <property name="text">
      <string>evo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="flowButton">
     <property name="text">
      <string>flow</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cacheLightButton">
     <property name="text">
      <string>cacheLight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="userViewButton">
     <property name="text">
      <string>userView</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
